packages/bpk-component-scrollable-calendar/readme.md

```diff
@@ -8,22 +8,21 @@
 npm install bpk-component-scrollable-calendar --save-dev
 ```
 
+### BpkScrollableCalendarGrid
+
+The BpkScrollableCalendarGrid component displays a month as a table.
+Above this table it displays the month's name in a `BpkText`. When
+used with `BpkScrollableCalendarDate`, it only displays days from
+that specific month. It is built on top of BpkCalendarGrid and
+uses the same props.
+
+[Please refer to the props of BpkCalendarGrid here](https://backpack.github.io/components/calendar?platform=web#props).
```

Comment:
Instead of just pointing to the props section of the Calendar page, we can actually point to the relevant sections:
https://backpack.github.io/components/calendar/?platform=web#bpkcalendargrid